app/lib/features/tasks/actions/create_task.dart

```diff
@@ -24,9 +24,12 @@ Future<void> showCreateTaskBottomSheet(
     showDragHandle: true,
     useSafeArea: true,
     isScrollControlled: true,
-    builder: (context) => CreateTaskWidget(
-      taskList: taskList,
-      taskName: taskName,
+    builder: (context) => Padding(
+      padding: MediaQuery.of(context).viewInsets,
```

Comment:
curios ... why is that not needed in other cases? Is it because the object then pulls the auto-focus? But others should be doing that, too, no? I'd like to understand the cause here ...

Or should we wrap that in a safe-area?

Reply:
Don't know exact root cause. I also faced similar kind of issue where I developer createEditTitle common widget where it was working fine on some area and misbehave at some places.

Above code just fixed the issue earlier also so applied it here.

Reply:
I can also confirm that wrapping with `safe-area` doesn't fixed this.

Reply:
Hm... that is unsatisfying ... would be good to figure out the root cause at some point...

Reply:
Hmm true, let me check if I can figure out something